extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -664,12 +669,13 @@ export class AmpIframe extends AMP.BaseElement {
    * @private
    */
   updateSize_(height, width) {
-    if (!this.isResizable_) {
+    if (!this.isResizable_ && !this.hasErroredEmbedSize_) {
       this.user().error(
         TAG_,
         'Ignoring embed-size request because this iframe is not resizable',
         this.element
       );
+      this.hasErroredEmbedSize_ = true;
```

Comment:
```suggestion
    if (!this.isResizable_) {
      if (!this.hasErroredEmbedSize_) {
        this.user().error(
          TAG_,
          'Ignoring embed-size request because this iframe is not resizable',
          this.element
        );
        this.hasErroredEmbedSize_ = true;
      }
```